tests/testcfg.rs

```diff
@@ -81,6 +90,29 @@ impl TestCfg {
     }
 }
 
+fn setup_toolchain(fuelup_home_path: &Path, toolchain: &str) -> Result<()> {
+    let bin_dir = fuelup_home_path
+        .join("toolchains")
+        .join(toolchain)
+        .join("bin");
+    fs::create_dir_all(&bin_dir).expect("Failed to create temporary latest toolchain bin dir");
+
+    for bin in ALL_BINS {
+        fs::File::create(&bin_dir.join(bin))?;
+    }
+
+    Ok(())
+}
+
+fn setup_settings_file(settings_path: &Path, default_toolchain: &str) -> Result<()> {
```

Comment:
nit: we could make this function append `settings.toml` rather than having to do it every time this is called.

```suggestion
fn setup_settings_file(settings_dir: &Path, default_toolchain: &str) -> Result<()> {
    let settings_path = &settings_dir.join("settings.toml");
```

Reply:
Ah, this is way better :+1: thanks for the suggestion!